zha/application/platforms/fan/__init__.py

```diff
@@ -107,7 +107,7 @@ def speed_count(self) -> int:
         return int_states_in_range(self.speed_range)
 
     @functools.cached_property
-    def supported_features(self) -> int:
+    def supported_features(self) -> FanEntityFeature:
```

Comment:
The HA side used `value in supported_features` to check if a feature is supported. The `in` operator is only valid on `IntFlag` Enums, not on `int` itself. This is already typed like this for cover, climate, siren and other entity types.